File: module_book/src/main/java/debug/BookApplication.java
package debug;

import android.content.Intent;

import com.ebook.book.service.DownloadService;
import com.xrn1997.common.BaseApplication;


public class BookApplication extends BaseApplication {
    @Override
    public void onCreate() {
        super.onCreate();
        startService(new Intent(this, DownloadService.class));
    }
}

File: module_book/src/main/java/debug/BookApplication.kt
package debug

import android.content.Intent
import com.ebook.book.service.DownloadService
import com.xrn1997.common.BaseApplication

class BookApplication : BaseApplication() {
    override fun onCreate() {
        super.onCreate()
        startService(Intent(this, DownloadService::class.java))
    }
}
